INSTALL

```diff
@@ -62,8 +62,8 @@ Getting Qt5
 
 We use Qt5 in order to only maintain one UI across platforms.
 
-Qt5.9.1 is the oldest version supported if ONLY building Subsurface (desktop version)
-Qt5.11 is the oldest version supported if also building Subsurface-mobile (ios/android)
+Qt5.9.1 is the oldest version supported if ONLY building Subsurface
+Qt5.12.5 is the oldest version supported if also building Subsurface-mobile
```

Comment:
since I am building the Android binaries against 5.12.0 right now I believe that this statement is incorrect, though